topics/transcriptomics/tutorials/clipseq/tutorial.md

```diff
@@ -0,0 +1,629 @@
+> 2. View the output image.
+> ![fingerprint](../../images/clipseq_fingerprint.png "Graph of IP strength from <b>plotFingerprint</b>..")
+>
+{: .hands_on}
+
+> ### {% icon question %} Questions
+>
+>    > 1. What does this graph represent?
+>    > 2. How do (or should) input datasets differ from CLIP datasets?
+>    > 3. What do you think about the quality of the CLIP experiment?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1. It shows us how good the CLIP Signal compared to the control signal is. Now be careful, CLIP-Seq experiments involve either a total RNA control or a negative control with another protein that unspecifically binds RNA (e.g., IgG). An ideal total RNA control (input control) like ours with perfect uniform distribution of reads along the genome/transcriptome (i.e. without enrichments) and infinite sequencing coverage should generate a straighter, diagonal line. On the other hand, a very specific and strong CLIP enrichment will be indicated by a prominent and steep rise of the cumulative sum towards the highest rank. Yet, a negative control often has the same sharp slope at the end as a CLIP experiment but often depicts a straighter, diagonal line in the beginning like the input control.
+> > 2. We expect that the control (input) has a different, more straighter, diagonal slope than the CLIP experiment.
+> > 3. Both the CLIP experiment and the control are closely related. It is maybe wise to check the correlation of the two samples.
+> {: .solution}
+>
+{: .question}
+
+For additional information on how to interpret **plotFingerprint** plots, read the information [here](https://deeptools.readthedocs.io/en/latest/content/tools/plotFingerprint.html#background).
+
+## Correlation between Samples
+
+We have seen that our input control and our CLIP experiment might be strongly correlated, meaning, that our potential RBFOX2 binding regions are not truly enrichment when comparing to a our control.
+We are therefore going to further check the correlation between our control and CLIP experiment.  
+
+> ### {% icon hands_on %} Hands-on: Assessing correlation between samples
+>
+> 1. **multiBamSummary** {% icon tool %} with the following parameters:
+>    - *"Sample order matters"*: `No`
+>    - *"Bam file"*: `both bam files` (output of **UMI-tools deduplicate** {% icon tool %})
+>    - *"Choose computation mode"*: `Bins`
+>     - *"Bin size in bp"*: `1000`
+>    - *"Show advanced options"*: `no`
+> 2. **plotCorrelation** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"Matrix file from the multiBamSummary tool"*: `outFile` (output of **multiBamSummary** {% icon tool %})
+>    - *"Correlation method"*: `Spearman`
+>    - *"Plotting type"*: `Heatmap`
+>    - *"Skip zeros"*: `Yes`
+>    - *"Image file format"*: `png`
+> 3. View the output image.
+> ![correlation](../../images/clipseq_correlation.png "Heatmap of correlation matrix generated by <b>plotCorrelation</b>.")
```

Comment:
same here please explain what people see and what is expected, what is bad data and so on.